src/cosmos/Swap.fixture.tsx

```diff
@@ -19,6 +19,21 @@ import EventFeed, { Event } from './EventFeed'
 import useOption from './useOption'
 import useProvider, { INFURA_NETWORK_URLS } from './useProvider'
 
+const TOKEN_WITH_NO_LOGO = {
+  chainId: 1,
+  decimals: 18,
+  symbol: 'HDRN',
+  name: 'Hedron',
+  address: '0x3819f64f282bf135d62168C1e513280dAF905e06',
+}
+
+const newTokenList = (tokens as TokenInfo[]).concat([TOKEN_WITH_NO_LOGO])
+const mainnetTokens = tokens.filter((token) => token.chainId === SupportedChainId.MAINNET)
+const tokenLists: Record<string, TokenInfo[]> = {
+  Default: newTokenList,
```

Comment:
The Default list should remain all `tokens`.
Can you make a new list, `Hedron`, or `Logoless`, and put this there?